examples/_client.rb.example

```diff
@@ -36,3 +36,5 @@ puts "Connected to OneView appliance at #{@client.url}\n\n"
 # @firmware_bundle_path = 'hp-firmware-a1b08f8a6b-HPGH-1.1.i386.rpm'
 # @sas_interconnect_type = 'Synergy 12Gb SAS Connection Module'
 # @synergy_enclosure_hostname = 'fe80::2:0:9:1%eth2'
+# @enclosure_name = '0000A66101'
+# @interconnect_name = '0000A66101, interconnect 3'
```

Comment:
Yes, it is possible determine the correct data that I want work with example, but is more complex that simply get by name. I thought about using variables because is more simple to get the data, but if it is better keep less variables even that example become a litle more complex, no problem for me. =)